### DISCUSSION TITLE
Sentry reports TypeError t is not a function form next/dist/compiled/web-vitals/web-vitals.es5.umd.min.js

### DISCUSSION THREAD

**Discussion by afurm** [vercel/next.js#55218] [Help]
### Summary

1. Deploy Next js App to Vercel
2. Go to Sentry sever and see Error:

```
TypeError (next/dist/compiled/web-vitals/web-vitals.es5.umd)
t is not a function. (In 't()', 't' is undefined)
```

This is happen on:
1.  Mobile Safari UI/WKWebView
2. Phone
3. IOS 16.6.1

We can't reproduce it locally.

The issue comes from `web-vitals`  but we didn’t have it in our package.json file. It seems that it's installed by Vercel when we do deployment .

### Additional information

We started to see this issue on Jun 20 2023 (3 month ago) we didn’t see it before and happens every day for a few times.
This is only reproducible on production when we deploy our app to Vercel. 

Our nextjs versions:
```
"next": "11.0.1",
"next-i18next": "^11.3.0",
"next-seo": "4.19.0",
```

**Comment by toniebe** [comment]
did you find a solution for the issue?

**Reply by sneko** [reply]
@afurm @toniebe , on my side it was due to using a deprecated integration for Sentry:
```ts
import { Offline as OfflineIntegration } from '@sentry/integrations';
```

After removing that it's good. Note that `t` is a minified name, so it can comes from whatever compiled library...
